src/scripts/OSFramework/OSUI/Pattern/Video/IVideo.ts

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Patterns.Video {
+	/**
+	 * Defines the interface for OutSystemsUI Video Pattern
+	 *
+	 * @export
+	 * @interface IVideo
+	 * @extends {Interface.IPattern}
+	 * @extends {Interface.ICallback}
+	 */
+	export interface IVideo extends Interface.IPattern {
+		registerCallback(eventName: string, callback: GlobalCallbacks.OSGeneric): void;
```

Comment:
This registerCallback should be removed, it's already defined on the IPattern

Reply:
Probably some leftovers from when the registerCakllbacks were move to the Abstract